tests/test_all_formats.py

```diff
@@ -17,7 +17,7 @@ def testHDF(self):
     def testDRS(self):
         f = self.getDataFile("dvtest1.dic")
         data = f['a']
-        self.assertEqual(data.missing_value, 1e20)
+        self.assertEqual(data.time, '15:17:00')
```

Comment:
I asked Ken about that,  do we need a `missing_value` for a DRS file?  It was not in the original file and I added it.  Doing that create `missing_value` for `_bounds` variable which is not `CF-1` compliant.